sql/updates/world/3.3.5/9999_99_99_99_world_burst.sql

```diff
@@ -0,0 +1,27 @@
+UPDATE `spell_script_names` SET `ScriptName`='spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59576' WHERE `spell_id` = 59576;
+UPDATE `spell_script_names` SET `ScriptName`='spell_q12690_burst_at_the_seams_52510' WHERE `spell_id` = 52510;
+
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (52593, 59579, 76245);
```

Comment:
always delete by name, not spellid.

Reply:
always? and if the name was 'spell_gen_creature_permanent_feign_death' which is shared by 10 spell_ids?
There are 252 shared spell names, "ALWAYS delete on ScriptName" is crazy.

Reply:
https://github.com/TrinityCore/TrinityCore/pull/22404#discussion_r224953307